erdpy/tests/test_cli_contracts.sh

```diff
@@ -60,6 +61,19 @@ testRunMandos() {
     ${ERDPY} --verbose contract test --directory="mandos" ${SANDBOX}/myfunding-rs || return 1
 }
 
+testWasmName() {
```

Comment:
:+1: (the name of the test is a bit unspecific / ambiguous, but it's a good test)